Wrapping/Generators/Python/itkExtras.py

```diff
@@ -642,8 +642,12 @@ def imread(filename, pixel_type=None, fallback_only=False):
 
     The reader is instantiated with the image type of the image file if
     `pixel_type` is not provided (default). The dimension of the image is
-    automatically found. If the given filename is a list or a tuple, the
-    reader will use an itk.ImageSeriesReader object to read the files.
+    automatically found. If the filename provided is a directory then the
```

Comment:
*If the filename* -> new paragraph that focuses on the DICOM series support.

Also, describe the condition that one series (and only one series) is expected in the directory. If more than one series is in the directory ->

Reply:
Will do.